test/fixtures/queries/python.py

```diff
@@ -0,0 +1,7 @@
+def foo(): pass
+# <- definition.function
+
+def bar():
```

Comment:
Did this file end up getting used anywhere?

Reply:
Ah whoops, a testing file that crept in. Removing.